arrows/core/mesh_intersect.cxx

```diff
@@ -0,0 +1,385 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
+ * \file
+ * \brief Implementation of mesh operations
+ */
+
+#include "mesh_intersect.h"
+#include <limits>
+#include <vital/logger/logger.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace core {
+
+using namespace kwiver::vital;
+
+/// Intersect the ray from point to a triangle
```

Comment:
Separator lines would be nice. Also, duplicating documentation from the header is discouraged.